Symfony/CS/Fixer.php

```diff
@@ -158,9 +167,51 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
         return $changed;
     }
 
-    public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileCacheManager $fileCacheManager)
+    private function getNextFile(\Iterator $iterator, FileCacheManager $fileCacheManager)
```

Comment:
this is not really getting the next file. It is getting the current file of the iterator (your code gets the next file because it moves the iterator before calling it)